angular-ngrx-scss/src/app/repository/components/repo-header/repo-header.component.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+
+@Component({
+  selector: 'app-repo-header',
+  templateUrl: './repo-header.component.html',
+  styleUrls: ['./repo-header.component.scss'],
+})
+export class RepositoryHeaderComponent implements OnInit {
+  @Input() owner = '';
+  @Input() name = '';
+  basePath = '';
+
+  ngOnInit(): void {
+    this.basePath = `/${this.owner}/${this.name}`;
```

Comment:
I wonder whether it may be better to make the basePath a getter instead. That way it would update properly even if we change the owner or name on runtime (I know that'll most likely not happen, but still... 😅 )

Alternative option: make "owner" and "name" into setters and update the basePath there.